readthedocs/api/v3/views.py

```diff
@@ -414,3 +430,36 @@ def get_view_name(self):
 
 class OrganizationsProjectsViewSet(SettingsOverrideObject):
     _default_class = OrganizationsProjectsViewSetBase
+
+
+class RemoteRepositoryViewSet(APIv3Settings, ListModelMixin, GenericViewSet):
+    model = RemoteRepository
+    serializer_class = RemoteRepositorySerializer
+    filterset_class = RemoteRepositoryFilter
+    queryset = RemoteRepository.objects.all()
+    permission_classes = (IsAuthenticated,)
+
+    def get_queryset(self):
+        queryset = super().get_queryset().api(self.request.user).annotate(
+            _admin=Exists(
+                RemoteRepositoryRelation.objects.filter(
+                    remote_repository=OuterRef('pk'),
+                    user=self.request.user,
+                    admin=True
+                )
+            )
+        )
+        return queryset.select_related('organization').order_by(
+            'organization__name', 'full_name'
+        ).distinct()
+
+
+class RemoteOrganizationViewSet(APIv3Settings, ListModelMixin, GenericViewSet):
+    model = RemoteOrganization
+    serializer_class = RemoteOrganizationSerializer
+    filterset_class = RemoteOrganizationFilter
+    queryset = RemoteOrganization.objects.all()
+    permission_classes = (IsAuthenticated,)
+
+    def get_queryset(self):
+        return super().get_queryset().api(self.request.user)
```

Comment:
I'd create a `RemoteQuerySetMixin` in `api/v3/mixins.py` with this line, and use it in both views. We are doing that for the other views.

`RemoteRepositoryViewSet` may need to overwrite it tho, to add the `.annotate`, but that's fine.